generator.py

```diff
@@ -171,7 +171,7 @@ def main():
         conf["gir_path"] = "./gir/target/release/gir"
 
     print('=> Regenerating crates...')
-    if not regen_crates(".", conf):
+    if not regen_crates("gtk4", conf):
```

Comment:
I don't think this change is needed?

Reply:
Oh wow, no! that was just for speeding things up. Will add another MR to make this an option.